### PR TITLE
Tackle-873 Seeding instances

### PR DESCRIPTION
**Pull request by CarmiWisemon**
MTA 6.0 

Resolves https://issues.redhat.com/browse/TACKLE-873

Preview https://deploy-preview-624--windup-documentation.netlify.app/docs/web-console-guide/master/index.html#mta-web-seeding-instances_user-interface-guide

### CODE REVIEW

File: docs/topics/mta-web-seeding-instances.adoc [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+Business services must be created before creating or importing applications.
+====
+
+There are no default business services defined and they must be created before creating the applications.
+
+You can follow these steps to create a new business service.
+
+.Procedure
+
+. In *Developer* view, click *Controls*.
```
Comment (rromannissen) [inline docs/topics/mta-web-seeding-instances.adoc:99]: This is the procedure to create tags, not Business Services.

File: docs/topics/mta-web-seeding-instances.adoc [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+** Rank - the order which the tags appear on the applications
+** Color
+. Click *Create* button.
+
+== Creating new tag types
+
+{ProductName} ({ProductShortName}) uses tags to classify applications in multiple categories. Follow the steps below to create a new tag type, which is not in the default list.
+
+.Procedure
+
```
Comment (rromannissen) [inline docs/topics/mta-web-seeding-instances.adoc:113]: This is the procedure to create Tags, not Tag Types

File: docs/topics/mta-web-seeding-instances.adoc [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+** Name
+** Rank - the order which the tags appear on the applications
+** Color
+. Click *Create*.
+
+=== Creating new tags
+
+You can follow these steps to create a new tag, which is not in the default list.
+
+.Procedure
```
Comment (rromannissen) [inline docs/topics/mta-web-seeding-instances.adoc:127]: This is the procedure to create Tag Types, not Tags

**Review by rromannissen — CHANGES_REQUESTED** [review]
Looks good overall, but there are some procedures that have been swapped for some reason.

File: docs/topics/mta-web-seeding-instances.adoc (hunk 1 above)
Comment (CarmiWisemon) [reply docs/topics/mta-web-seeding-instances.adoc:99]: My mistake. I corrected it. Thanks

File: docs/topics/mta-web-seeding-instances.adoc (hunk 2 above)
Comment (CarmiWisemon) [reply docs/topics/mta-web-seeding-instances.adoc:113]: I took the information from here, https://github.com/konveyor/konveyor.github.io/blob/main/content/Tackle/instances.md

1. Should Creating New Tag Types come before Creating New Tags? 

2. Both "Creating New Tags" and "Creating New Tag Types" have a step called - Click "Create tag types". 
Does that step belong in both procedures?
What is the correct procedure for Creating Tags and Creating Tag Types?

Thanks!

File: docs/topics/mta-web-seeding-instances.adoc (hunk 3 above)
Comment (CarmiWisemon) [reply docs/topics/mta-web-seeding-instances.adoc:127]: See above

File: docs/topics/mta-web-seeding-instances.adoc (hunk 2 above)
Comment (CarmiWisemon) [reply docs/topics/mta-web-seeding-instances.adoc:113]: Corrected based on SCoons feedback.

File: docs/topics/mta-web-seeding-instances.adoc (hunk 3 above)
Comment (CarmiWisemon) [reply docs/topics/mta-web-seeding-instances.adoc:127]: See above

**Review by sshveta — APPROVED** [review]

File: docs/topics/mta-web-seeding-instances.adoc [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+== Creating a new business service
+
+{ProductName} ({ProductShortName}) uses the business services attribute to define the departments within the organization that use the application and that are affected by the migration.
+
+[NOTE]
+====
+Business services must be created before creating or importing applications.
+====
+
+There are no default business services defined and they must be created before creating the applications.
```
Comment (kpunwatk) [inline docs/topics/mta-web-seeding-instances.adoc:93]: Hi @CarmiWisemon,
Business services is not a must/mandatory item while creating/importing an application.
It was before but not now.
Lines 90 and 93 not necessary

**Review by kpunwatk — CHANGES_REQUESTED** [review]

File: docs/topics/mta-web-seeding-instances.adoc [hunk 5]
```diff
@@ -0,0 +1,135 @@
+// Module included in the following assemblies:
+//
+// * docs/rules-development-guide/master.adoc
+
+:_content-type: PROCEDURE
+[id="mta-web-seeding-instances_{context}"]
+= Seeding instances
```
Comment (sbeskin-redhat) [inline docs/topics/mta-web-seeding-instances.adoc:7]: @CarmiWisemon 
I would suggest 'Seeding an instance'.

File: docs/topics/mta-web-seeding-instances.adoc [hunk 6]
```diff
@@ -0,0 +1,135 @@
...
+
+If you are a project architect you can configure the instances key parameters in the Controls window, before migration. The parameters can be added and edited as needed.
```
Comment (sbeskin-redhat) [inline docs/topics/mta-web-seeding-instances.adoc:9]: @CarmiWisemon 
"If you are a project architect, you..."
"...configure the instance's key parameters..."

File: docs/topics/mta-web-seeding-instances.adoc [hunk 7]
```diff
@@ -0,0 +1,135 @@
...
+The following parameters define applications, individuals, teams, verticals or areas within an organization affected or participating in the migration:
+
+* Stakeholders
+* Stakeholder groups
+* Job functions
+* Business services
+* Tag types
+* Tags
+
+You can create and configure an instance in any order. However, the suggested order below is the most efficient for creating stakeholders and tags.
```
Comment (sbeskin-redhat) [inline docs/topics/mta-web-seeding-instances.adoc:19]: @CarmiWisemon 
Why 'create'? The instance has already been created. I would suggest "You can configure an instance in any order."

File: docs/topics/mta-web-seeding-instances.adoc [hunk 8]
```diff
@@ -0,0 +1,135 @@
...
+. Create stakeholder groups
+. Create job functions
+. Create stakeholders
+
+Tags:
+
+. Create tag types
+. Create tags
+
+Stakeholders and defined by:
```
Comment (sbeskin-redhat) [inline docs/topics/mta-web-seeding-instances.adoc:32]: @CarmiWisemon 
Probably, "...are defined"?
"... by the following:"

File: docs/topics/mta-web-seeding-instances.adoc [hunk 9]
```diff
@@ -0,0 +1,135 @@
...
+
+* Email
+* Name
+* Job function
+* Stakeholder groups
+
+== Creating a new stakeholder group
+
+You can follow these steps to create a new stakeholder group. There are no default stakeholder groups defined.
```
Comment (sbeskin-redhat) [inline docs/topics/mta-web-seeding-instances.adoc:41]: @CarmiWisemon 
I would suggest:
"There are no default stakeholder groups. You can create a new stakeholder group by following the procedure below."

File: docs/topics/mta-web-seeding-instances.adoc [hunk 10]
```diff
@@ -0,0 +1,135 @@
...
+** Name
+** Description
+** Member(s)
+. Click *Create*.
+
+== Creating a new job function
+
+{ProductName} ({ProductShortName}) uses the job function attribute to classify stakeholders and provides a list of default values that can be expanded.
+
+You can follow these steps to create a new job function, which is not in the default list.
```
Comment (sbeskin-redhat) [inline docs/topics/mta-web-seeding-instances.adoc:58]: @CarmiWisemon 
Does this really mean that the user can add DEFAULT values? Probably not. Probably, it just means that new job functions can be created. 
I would suggest replacing the two paragraphs with this: 
"{ProductName} ({ProductShortName}) uses the job function attribute to classify stakeholders and provides a list of default values. You can create new job functions by following the procedure below.

File: docs/topics/mta-web-seeding-instances.adoc [hunk 11]
```diff
@@ -0,0 +1,135 @@
...
+
+. In *Developer* view, click *Controls*.
+. Click *Job functions*.
+. Click *Create new*.
+. Enter a job function title in the *Name* text box.
+. Click *Create*.
+
+== Creating a new stakeholder
+
+You can follow these steps to create a new migration project stakeholder.
```
Comment (sbeskin-redhat) [inline docs/topics/mta-web-seeding-instances.adoc:70]: @CarmiWisemon 
"You can create a new migration project stakeholder by following the procedure below."

File: docs/topics/mta-web-seeding-instances.adoc [hunk 12]
```diff
@@ -0,0 +1,135 @@
...
+
+.Procedure
+
+. In *Developer* view, click *Controls*.
+. Click *Stakeholders*.
+. Click *Create new*.
+. Enter the following information:
+** Email
+** Name
+** Job function - custom functions can be created
```
Comment (sbeskin-redhat) [inline docs/topics/mta-web-seeding-instances.adoc:80]: @CarmiWisemon 
"custom functions can be created" - why is it here? Creating new job functions has already been described above.

File: docs/topics/mta-web-seeding-instances.adoc [hunk 13]
```diff
@@ -0,0 +1,135 @@
...
+** Stakeholder group
+. Click *Create*.
+
+== Creating a new business service
+
+{ProductName} ({ProductShortName}) uses the business services attribute to define the departments within the organization that use the application and that are affected by the migration.
```
Comment (sbeskin-redhat) [inline docs/topics/mta-web-seeding-instances.adoc:86]: @CarmiWisemon 
"...business service attribute..."
"...to specify the departments..."

File: docs/topics/mta-web-seeding-instances.adoc [hunk 14]
```diff
@@ -0,0 +1,135 @@
...
+{ProductName} ({ProductShortName}) uses tags to classify applications in multiple categories. Follow the steps below to create a new tag type, which is not in the default list.
+
+.Procedure
+
+. In *Developer* view, click *Controls*.
+. Click *Tags*.
+. Click *Create tag type*.
+. Enter the following information:
+** Name
+** Rank - the order which the tags appear on the applications
```
Comment (sbeskin-redhat) [inline docs/topics/mta-web-seeding-instances.adoc:119]: @CarmiWisemon 
"...the order in which..."

File: docs/topics/mta-web-seeding-instances.adoc (hunk 5 above)
Comment (CarmiWisemon) [reply docs/topics/mta-web-seeding-instances.adoc:7]: Thank you!

File: docs/topics/mta-web-seeding-instances.adoc (hunk 9 above)
Comment (CarmiWisemon) [reply docs/topics/mta-web-seeding-instances.adoc:41]: Thank you!

File: docs/topics/mta-web-seeding-instances.adoc [hunk 15]
```diff
@@ -0,0 +1,135 @@
...
+** Color
+. Click *Create*.
+
+=== Creating new tags
+
+You can follow these steps to create a new tag, which is not in the default list.
```
[the remark this answers is not in the document]
Comment (CarmiWisemon) [reply docs/topics/mta-web-seeding-instances.adoc:125]: Thank you.

File: docs/topics/mta-web-seeding-instances.adoc (hunk 4 above)
Comment (CarmiWisemon) [reply docs/topics/mta-web-seeding-instances.adoc:93]: Updated.

File: docs/topics/mta-web-seeding-instances.adoc (hunk 10 above)
Comment (sbeskin-redhat) [reply docs/topics/mta-web-seeding-instances.adoc]: @CarmiWisemon 
Why do you want to explain that the new job function is not in the default list?  'New' already means that it is not default. Also, it should be 'that', not 'which' and without the comma.

File: docs/topics/mta-web-seeding-instances.adoc [hunk 16]
```diff
@@ -0,0 +1,135 @@
...
+. Click *Create new*.
+. Enter the following information:
+** Name
+** Description
+** Owner
+. Click *Create*.
+
+== Creating new tag types
+
+{ProductName} ({ProductShortName}) uses tags to classify applications in multiple categories. Follow the steps below to create a new tag type, which is not in the default list.
```
[the remark this answers is not in the document]
Comment (sbeskin-redhat) [reply docs/topics/mta-web-seeding-instances.adoc]: @CarmiWisemon 
Why do you want to explain that the new tag type is not in the default list?  'New' already means that it is not default. Also, it should be 'that', not 'which' and without the comma.

**Pull request merged**